state_machine/drivers/pycubedmini/lib/pycubed.py

```diff
@@ -63,17 +63,25 @@ class _Satellite:
     # Define NVM flags
     f_contact = bitFlag(register=_FLAG, bit=1)
     f_burn = bitFlag(register=_FLAG, bit=2)
-    f_free1 = bitFlag(register=_FLAG, bit=3)
-    f_free2 = bitFlag(register=_FLAG, bit=4)
 
     # Define NVM counters
-    c_boot = multiBitFlag(register=_BOOTCNT, lowest_bit=0, num_bits=8)
+    c_boot = multiByte(num_bytes=2, lowest_register=_BOOTCNT)
     c_state_err = multiBitFlag(register=_RSTERRS, lowest_bit=4, num_bits=4)
     c_vbus_rst = multiBitFlag(register=_RSTERRS, lowest_bit=0, num_bits=4)
     c_deploy = multiBitFlag(register=_DCOUNT, lowest_bit=0, num_bits=8)
     c_downlink = multiBitFlag(register=_DWNLINK, lowest_bit=0, num_bits=8)
     c_logfail = multiBitFlag(register=_LOGFAIL, lowest_bit=0, num_bits=8)
 
+    # Define maxvals
+    max_vals = {
```

Comment:
I don't think this dictionary should be in the driver, this data is already accessible in a more reasonable manner.

Reply:
This isn't available calling `cubesat.c_boot.maxval` (I don't entirely understand why, @yashika-batra  explained it to me, and I tried this yesterday.)